test/test_retry.py

```diff
@@ -155,6 +155,27 @@ def test_backoff(self) -> None:
 
         assert retry.get_backoff_time() == max_backoff
 
+    def test_configurable_backoff_max(self) -> None:
+        """ Configurable backoff is computed correctly """
+        max_backoff = 1
+
+        retry = Retry(total=100, backoff_factor=0.2, backoff_max=max_backoff)
+        assert retry.get_backoff_time() == 0  # First request
+
+        retry = retry.increment(method="GET")
+        assert retry.get_backoff_time() == 0  # First retry
+
+        retry = retry.increment(method="GET")
+        assert retry.backoff_factor == 0.2
+        assert retry.total == 98
+        assert retry.get_backoff_time() == 0.4  # Start backoff
+
+        retry = retry.increment(method="GET")
+        assert retry.get_backoff_time() == 0.8
+
+        retry = retry.increment(method="GET")
+        assert retry.get_backoff_time() == max_backoff
```

Comment:
Should add an additional case that after another `.increment()` our `get_backoff_time()` is still `max_backoff`

Reply:
Added